bigtable-client-core/src/main/java/com/google/cloud/bigtable/config/BigtableOptions.java

```diff
@@ -206,14 +205,6 @@ private BigtableOptions(
       RetryOptions retryOptions,
       int timeoutMs,
       int channelCount) {
-    Preconditions.checkArgument(
-        !Strings.isNullOrEmpty(projectId), "ProjectId must not be empty or null.");
```

Comment:
The BigtableClusterName is no longer created unless all of the values are set.  I added some tests to make sure that the expected behavior happens.